benchmarks/benchmark/tools/model-analyzer/README.md

```diff
@@ -0,0 +1 @@
+In the future, this directory will contain the Triton model analyzer related benchmarks.
```

Comment:
Same here comment here, can we just add the diretory later when Triton related benchmarks are available?

Reply:
Done